lib/rspecq/reporter.rb

```diff
@@ -83,6 +83,13 @@ def summary(failures, errors, flaky_jobs, duration)
       end
 
       summary = ""
+      if @queue.build_failed_fast?
+        summary << "\n\n"
+        summary << "The limit of #{@queue.fail_fast} failures has been reached\n"
+        summary << "Aborting..."
+        summary << "\n"
+      end
+
       summary << failed_examples_section if !failures.empty?
 
       errors.each { |_job, msg| summary << msg }
```

Comment:
I'm curious: at which point is the `Reporter` signaled to exit due to a failed-fast build?

I believe the current code doesn't work as expected (another hint that we really need tests in the reporter).